### DISCUSSION TITLE
[LUA] about commands Table

### DISCUSSION THREAD

**Discussion by Bradly7412** [YimMenu/YimMenu#2944] [Q&A]
Does the `command.call("waypointtp", {})` stop velocity? if so is there a way around it with/without using it?

**Comment by xesdoog** [comment]
Do you mean if you teleport while moving you stop? If so then setting coords in GTA always stops the entity. You'll have to make your own Lua with a regular teleport function but register your velocity or forward speed before setting new coords then add it back after.
